components/support/rc_crypto/nss/src/util.rs

```diff
@@ -3,184 +3,165 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use crate::error::*;
-use crate::ErrorKind::NSSUninitialized;
 use nss_sys::*;
-use std::{ffi::CString, os::raw::c_char, sync::Once};
+use std::{ffi::CString, os::raw::c_char};
+
+use std::path::PathBuf;
+use std::sync::OnceLock;
 
 #[cfg(feature = "keydb")]
 use crate::pk11::slot;
-#[cfg(feature = "keydb")]
-use once_cell::sync::OnceCell;
-#[cfg(feature = "keydb")]
-use std::{fs, path::Path};
 
 // This is the NSS version that this crate is claiming to be compatible with.
 // We check it at runtime using `NSS_VersionCheck`.
 pub const COMPATIBLE_NSS_VERSION: &str = "3.26";
 
-static NSS_INIT: Once = Once::new();
-#[cfg(feature = "keydb")]
-static NSS_PROFILE_PATH: OnceCell<String> = OnceCell::new();
-
-// Check whether either thr Rust-internal NSS initialization has run, or NSS has been initialized
-// in the outside.
-#[cfg(not(feature = "keydb"))]
-fn is_nss_initialized() -> bool {
-    NSS_INIT.is_completed() || unsafe { NSS_IsInitialized() == PR_TRUE }
+pub fn expect_nss_initialized() -> Result<()> {
+    INITIALIZED
+        .get()
+        .map(|_| {})
```

Comment:
nit: `.map(|_| ())` is more idiomatic IMO.

Reply:
thanks!